inlong-dataproxy/dataproxy-source/src/main/java/org/apache/inlong/dataproxy/utils/MessageUtils.java

```diff
@@ -56,33 +63,197 @@ public static boolean isSyncSendForOrder(Event event) {
     }
 
     /**
-     * Convert String to ByteBuf
+     *  process response to client
+     * @param commonAttrMap attribute map
+     * @param resultMap     result map
+     * @param remoteChannel client channel
+     * @param msgType       the message type
+     */
+    public static void returnSourceRspPackage(Map<String, String> commonAttrMap,
```

Comment:
suggest to remove "return" in the method name

Reply:
Ok